.github/workflows/deploy-release.yml

```diff
@@ -70,16 +70,18 @@ jobs:
         timeout-minutes: 25
         env:
           ORG_GRADLE_PROJECT_IS_SNAPSHOT: false
+          ORG_GRADLE_PROJECT_NATIVE_TARGETS_ENABLED: false
           ORG_GRADLE_PROJECT_ZCASH_ASCII_GPG_KEY: ${{ secrets.MAVEN_SIGNING_KEY_ASCII }}
         run: |
           ./gradlew publishAllPublicationsToMavenLocalRepository --no-parallel
       - name: Deploy to Maven Central
         timeout-minutes: 5
         env:
-          ORG_GRADLE_PROJECT_ZCASH_MAVEN_PUBLISH_USERNAME: ${{ secrets.MAVEN_CENTRAL_USERNAME }}
-          ORG_GRADLE_PROJECT_ZCASH_MAVEN_PUBLISH_PASSWORD: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
-          ORG_GRADLE_PROJECT_ZCASH_ASCII_GPG_KEY: ${{ secrets.MAVEN_SIGNING_KEY_ASCII }}
           ORG_GRADLE_PROJECT_IS_SNAPSHOT: false
+          ORG_GRADLE_PROJECT_NATIVE_TARGETS_ENABLED: false
```

Comment:
In addition to adding `ORG_GRADLE_PROJECT_NATIVE_TARGETS_ENABLED`, I sorted the rest of the values here.